blueoil/generate_lmnet_config.py

```diff
@@ -296,10 +297,12 @@ def _save(lmnet_config):
     tpl = env.get_template(template_file)
 
     applied = tpl.render(lmnet_config)
-    config_file = "{}.py".format(lmnet_config['model_name'])
-    with open(config_file, 'w') as fp:
+    with NamedTemporaryFile(
+            prefix="blueoil_config_{}".format(lmnet_config['model_name']),
+            suffix=".py", delete=False, mode="w") as fp:
         fp.write(applied)
-    return config_file
+        print(fp.name)
```

Comment:
This line `print(fp.name)` looks only for debug.
Please delete 👍

Reply:
Good catch. thanks!